collect_app/src/main/java/org/odk/collect/android/map/GoogleMapFragment.java

```diff
@@ -493,6 +496,9 @@ protected Marker createMarker(GoogleMap map, MapPoint point, boolean draggable)
     }
 
     protected BitmapDescriptor getBitmapDescriptor(int drawableId) {
+        if (testMode) {
+            return null; // during Robolectric tests, BitmapDescriptorFactory doesn't work
```

Comment:
This is unfortunate. Do you think it would be worth stubbing out `BitmapDescriptorFactory` so you don't need to include test logic in the real class?

Reply:
It turns out this guard actually isn't needed anymore—tests don't go through this code path because `map` is `null`.

It would be nice to be able to fix this with little stubs, but really the bigger issue is that we can't get `GoogleMap` to work at all under Robolectric.  Since the whole `map` is `null` there's a lot of other stuff that just has to be skipped in tests.  (Lots of people have complained about this—writing a stub for `GoogleMap` would be nice too but it's a bigger project.)

Anyway, I was at least able to take out these two lines, so there's less testing cruft in here.